src/app/shared/example/example.component.scss

```diff
@@ -8,6 +8,7 @@
     padding: 0;
     margin: 0 10px 0 10px;
     width: calc(100% - 20px);
+    height: 50px;
```

Comment:
why this height? I think this component height should be flexible but max-height:64px